server/src/controllers/maproom/v2/cells/wildMonsterCell.ts

```diff
@@ -1,31 +1,41 @@
+import { WorldMapCell } from "../../../../models/worldmapcell.model";
+import { ORMContext } from "../../../../server";
+import { Save } from "../../../../models/save.model";
+
 const tribes = ["Kozu", "Legionnaire", "Dreadnaut", "Abunakki"];
 
-export const wildMonsterCell = async () => {
+export const wildMonsterCell = async (cell?: WorldMapCell) => {
   const randomIndex = Math.floor(Math.random() * tribes.length);
   const tribeName = tribes[randomIndex];
 
+  const save = await ORMContext.em.findOne(Save, {
+    basesaveid: cell?.base_id
+  })
+
   return {
-    uid: 0,
+    cell_id: cell?.cell_id || 0,
```

Comment:
Reminder: We should remove this, and only pass the fields the client expects (not new ones such as `cell_id`